packages/pds-ember/app/styles/pds/theme/_light-deprecated.scss

```diff
@@ -31,3 +31,4 @@ $word-spacing: $wordSpacing;
 $border-color-l1: $borderColor--l1;
 $border-color: $borderColor;
 $border-radius: $borderRadius;
+$fontSize--ui: $ui-fontSize;
```

Comment:
Why add this in the deprecated file?

Reply:
Aha, it's in prep for 14px removal 🔪

Reply:
Because `$ui-fontSize` follows the new experimental naming scheme of component-specific variables, but `$fontSize--ui` does not.

I was finding that it was difficult to organize properties by `propName--variantOrComponent` naming convention.